_sass/content.scss

```diff
@@ -112,3 +113,29 @@
     font-weight: 500;
   }
 }
+
+#sidebar.active {
+  display: none;
+}
+@media only screen and (min-width: 739px) {  
+  #main-content.moved {
+    margin-left: calc((100% - 739px) / 2 + -80px);    
+  }
+  #headerid.moved {
+    margin-left: calc((100% - 739px) / 2 + 65px);    
+  }
+  #sidebarCollapse.moved {
+    margin-left: calc((100% - 739px) / 2 + -80px);   
+  } }
+@media only screen and (max-width: 739px) {
+  #sidebarCollapse {
+    display: none;    
+  } }
+#sidebarCollapse.new.colored {
+  background-color: $base-button-color;
+  color: $link-color;
+}
+#sidebarCollapse.new {
+  background-color: $link-color;
+  color: white;
```

Comment:
> Color `white` should be written in hexadecimal form as `#ffffff`
> Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.
> Simply use the keyword "white" wherever necessary
@aayush-05 asked me to use keyword instead of hex

Reply:
Use the sass variable $white. It is already declared in the code. @namantechlabs